src/main/java/bio/terra/pdao/bigquery/BigQueryPdao.java

```diff
@@ -400,6 +397,137 @@ public void addRowIdsToStagingTable(Dataset dataset, String stagingTableName) {
         bigQueryProject.query(sql.toString());
     }
 
+    public List<String> getOverLappingRows(Dataset dataset,
```

Comment:
Ben and I talked in-person, but for other people looking at this: the APIs will be reworked so they don't try to pull all the row IDs into memory as a list.